bedrock/firefox/templates/firefox/hub/home.html

```diff
@@ -41,7 +36,9 @@
         <div class="header-container">
           <div class="header-content">
             <h1 class="page-title"><span>{{ _('Firefox') }}</span></h1>
+            {# L10n: this is the main page heading shown underneath the Firefox logo. #}
             <p class="tagline"><span>{{ _('The 100% fresh, free-range, ethical browser') }}</span></p>
+            {# L10n: here is an alternative page heading should the main heading not translate easily: "Browse like you’re meant to...fast, free and privately." #}
```

Comment:
This doesn't work, the comment is associated to the next string extracted. You should merge it with the comment before.